scripts/image_functions.sh

```diff
@@ -2,13 +2,14 @@
 export PYTHONIOENCODING
 
 function cleanup() {
-	find . -name "${IMAGE_NAME}" -type d -exec rm -rf '{}' \;
-	# delete any orphaned temporary images
+	find . -name "${IMAGE_NAME}*.d" -type d -exec rm -rf {} +
+    find . -name "${IMAGE_NAME}*.qcow2" -type f -exec rm -f {} +
```

Comment:
Indentation seems of. Urk, I reviewing 'rm -f' commands... ...so much power so much that can go wrong, but it seems okay.

Reply:
this is fust adding to what was there before.